File: PROBLEMS/TWO-SUM/soln1.js
var arr = [4, 5, 1, 7, 2];
var k = 13;
var twoSum = function (arr, k) {
    for (var i = 0; i < arr.length - 1; i++) {
        for (var j = i + 1; j < arr.length; j++) {
            var sum = arr[i] + arr[j];
            if (sum == k)
                return true;
        }
    }
    return false;
};
console.log(twoSum(arr, k));

File: PROBLEMS/TWO-SUM/soln1.rb
arr = [4, 5, 1, 7, 2]
k = 3

def two_sum(arr, k)
  for i in 0...arr.length-1
    for j in i+1...arr.length
      sum = arr[i] + arr[j]
      if sum == k
        return true
      end
    end
  end
  return false
end

puts two_sum(arr, k)
